doc/changelog.rst

```diff
@@ -217,11 +225,6 @@ Breaking Changes in 4.0
 - ``tz_aware``, an argument for :class:`~bson.json_util.JSONOptions`,
   now defaults to ``False`` instead of ``True``. ``json_util.loads`` now
   decodes datetime as naive by default.
-- ``directConnection`` URI option and keyword argument to :class:`~pymongo.mongo_client.MongoClient`
-  defaults to ``False`` instead of ``None``, allowing for the automatic
-  discovery of replica sets. This means that if you
-  want a direct connection to a single server you must pass
-  ``directConnection=True`` as a URI option or keyword argument.
```

Comment:
I think we should keep this in the list here too just for more visibility.

Reply:
Done.